docs/troubleshooting/large-partition-table.rst

```diff
@@ -85,6 +89,8 @@ Configure the detection thresholds of large partitions with the ``compaction_lar
 and the ``compaction_rows_count_warning_threshold`` paramater (default 100000)
 in the scylla.yaml configuration file.
 Partitions that are bigger than the size threshold and/or hold more than the rows count threshold are reported in the ``system.large_partitions`` table and generate a warning in the Scylla log (refer to :doc:`log </getting-started/logging/>`).
+The number of rows includes live rows as well as dead rows (that are either deleted or expired), and range tombstone.
```

Comment:
I think this sentence is simply not needed, it creates confusion.

Reply:
okay